docker/pull.bzl

```diff
@@ -18,6 +18,19 @@ Bazel rule for downloading base images without a Docker client to
 construct new images with docker_build.
 """
 
+def _python(repository_ctx):
+  if "BAZEL_PYTHON" in repository_ctx.os.environ:
+    return repository_ctx.os.environ.get("BAZEL_PYTHON")
+
+  python_path = repository_ctx.which("python")
+  if not python_path:
+    python_path = repository_ctx.which("python.exe")
+  if python_path:
+    return python_path
+
+  fail("rules_docker requires a python interpreter installed. " +
```

Comment:
Do we actually ever want to use system python? What if that's a python 3 executable, for example?

Reply:
I believe `repository_ctx.which` will return the system Python.  I think the answer is that we'll blow up (there is another issue on this).